codes/quantum/qudits/three_qutrit_code.yml

```diff
@@ -0,0 +1,91 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: three_qutrit_code
+physical: qutrits
+logical: qutrits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Three Qutrit Code'
+
+# First reference with the code (optional).
+introduced: '\cite{arXiv:quant-ph/9901025v1}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'The three qubtrit code is a quantum error correction code protecting a single qutrit encoded in three qutrits (qudit with \( d = 3 \) ) against a single erasure. It is an example of a \( ((n = 3, k = 2)) \) threshold scheme where a secret (quantum information) is split into \( n \) shares and can be reconstructed by \( k \) pieces. They key property of this code is that the reduced density matrix of any single qutrit is maximally mixed, meaning no information can be extracted from a single qutrit. Therefore, any arbitrary single qutrit tells you nothing about the encoded message, but access to any two pairs of qutrits will reveal the secret.'
```

Comment:
The code is easy enough to be able to define codewords explicitly, either as three equations, or if youd like one equation that contains a logical index. This might make some of the stuff below easier to describe.